### PR TITLE
Realistic

### PR DESCRIPTION
**Pull request by crmorford**
more tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,7 +11,7 @@ quicktests/overlaying/list_of_quicktests.json
 .idea/
 # Emacs
 .#*
-
+.ruby-version
```
Comment (bluong) [inline .gitignore]: Why this?

**Comment by bluong** [comment]
You're gonna want to merge develop.  It seems that it wants to bump the version number backwards

File: quicktests/template.html [hunk 2]
```diff
@@ -11,19 +11,70 @@
...
+        // key.on(68, function(keyData){
+        //     if(data.length > 0){
+        //       data.splice(data.length-1,1);
+        //       table.redraw();  
+        //     }     
+        // });
+          
+        plot.registerInteraction(hover);
+        // plot.registerInteraction(key);
+
```
Comment (bluong) [inline quicktests/template.html]: I'm not sure if this specific idea should used as the template

**Comment by bluong** [comment]
Run through the filenames so that they are as sensible as possible

File: .gitignore (hunk 1 above)
Comment (crmorford) [inline .gitignore]: UH idk i will fix that

File: quicktests/template.html (hunk 2 above)
Comment (crmorford) [inline quicktests/template.html]: woah i did not change this on purpose either. i need to figure out what happened.

**Comment by bluong** [comment]
Merge the branch again?  It seems that the diff will seemingly knock out the stuff that's currently in develop which is a bit concerning

**Comment by bluong** [comment]
This diff is really confusing...  It's claiming that you are adding stuff to the codebase that's already in existence and that you are also removing stuff from the codebase that shouldnt be removed

**Comment by crmorford** [comment]
ummm okay. let me deal with this when i have a big chunk of time. closing for now.

**Pull request closed without merging**